<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>butcheryandseafood</class>
 <widget class="QDialog" name="butcheryandseafood">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_butcherysearch">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>661</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_searchbutchery">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>70</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_productnamebutchery">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_butcheryproductprice">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_butcheryproductdesc">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_butcheryproductimage">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addbutcheryproduct">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_viewbutcheryproduct">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>View Cart</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
